scripts/extraction/utils.py

```diff
@@ -112,13 +118,60 @@ def parse(
         **(article_text_config if article_text_config else {}),
     )
 
+    # Update alias-entity probabilities (can only be done after parsing everything).
+    logger.info("Computing prior probabilities for alias-entity pairs.")
+    _update_prior_probs(db_conn, language)
+
+
+def _update_prior_probs(
```

Comment:
Collecting the data necessary for the computation of prior alias-entity probabilities turned out to be a bottleneck, hence we are computing them once after the extraction and store them in the DB.